### DISCUSSION TITLE
Redux toolkit in Next JS 13 with turbopack

### DISCUSSION THREAD

**Discussion by aprijal99** [reduxjs/redux-toolkit#3435] [Q&A]
Hi, I implemented redux toolkit in my Next JS 13 project, I was trying to use new Turbopack tool (even though it still beta) because found webpack really slow, but I got an error like the following image, has someone have any idea?

![redux-toolkit-error](https://github.com/reduxjs/redux-toolkit/assets/58645926/ee13ddcd-640f-4360-9554-a93925d2ea39)

**Comment by markerikson** [comment]
Is React-Redux installed?

**Reply by aprijal99** [reply]
Yes I installed @reduxjs/toolkit:1.9.3 and react-redux:8.0.5. The application ran successfully with webpack but got an error with turbopack

**Reply by markerikson** [reply]
Unfortunately, I don't think we have any idea what's going on here.

This is likely some kind of a Next/Turbopack bug.  I'd suggest filing an issue over there.

**Reply by aprijal99** [reply]
Yeah I think so, because I also tried implementing redux toolkit in fresh nextjs 13, but got the same error when using the turbopack

Thank you @markerikson
